kodilla-testing/src/main/java/com/kodilla/testing/shape/Circle.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.testing.shape;
+
+class Circle implements Shape{
+
+    private String shapeName;
```

Comment:
Ja bym pola shapeName nie trzymał w obiekcie, tylko zrobił:
public String getShapeName() { return this.getClass().getSimpleName(); }